assets/src/edit-story/components/keyboardShortcutsMenu/index.js

```diff
@@ -50,23 +51,33 @@ function KeyboardShortcutsMenu() {
   const [isOpen, setIsOpen] = useState(false);
 
   const closeMenu = useCallback(() => setIsOpen(false), [setIsOpen]);
+  const {
+    close: closeHelpCenter,
+  } = useHelpCenter(({ actions: { close } }) => ({ close }));
 
-  const toggleMenu = useCallback((e, showMenu) => {
-    e.preventDefault();
-    setIsOpen((prevIsOpen) => {
-      const menuOpen = showMenu ?? !prevIsOpen;
+  const toggleMenu = useCallback(
+    (e, showMenu) => {
+      e.preventDefault();
+      setIsOpen((prevIsOpen) => {
+        const menuOpen = showMenu ?? !prevIsOpen;
 
-      if (isKeyboardUser() && !menuOpen) {
-        // When menu closes, return focus to toggle menu button
-        anchorRef.current.focus?.();
-      }
+        if (menuOpen) {
+          closeHelpCenter();
+        }
```

Comment:
@embarks, thanks - this was suddenly very easy to add!

(and thanks @littlemilkstudio for pointing it out)

Reply:
Nice! I'm glad :D